### ISSUE TITLE
pingInterval on createSocket

### ISSUE THREAD

**Issue by davedoesdev** [primus/primus#579]
**Version:** 7.0.1

**Environment:**
  - **Operating system:** Ubuntu 16.10
  - **Browser:** N/A
  - **Node.js:** 7.8.0

**Expected result:** Process exits

**Actual result:** Process doesn't exit

**Steps to reproduce:**

- Use `Primus.createSocket()` to connect to a server on Node.js
- Kill the server
- The client doesn't exit because of:

```
# Timer
/home/david/centro/node_modules/primus/index.js:336  - this.heartbeatInterval = setInterval(
/home/david/centro/node_modules/primus/index.js:111  - this.initialise(options.transformer, options);
/home/david/centro/node_modules/primus/index.js:1097 - var primus = new Primus(new EventEmitter(), options);
```
If instead I do `Primus.createSocket({pingInterval: 0})` then the client exits when the server is killed.

Reason is when we do `Primus.createSocket`, the heartbeat interval is started. But the Primus instance is only used to get a socket, not as a server.

So my question is should we default `pingInterval` to `0` in `Primus.createSocket`?

**Comment by lpinca** [comment]
Yes, it makes sense, good catch. Not sure if it's better to use `false`?

**Comment by 3rd-Eden** [comment]
Fixed in #580
